tools/github-actions/yarn-errors-reporter/src/main.ts

```diff
@@ -50,9 +50,16 @@ async function run(): Promise<void> {
         }
       }
 
-      await exec('git', ['revert', '--no-commit', '-m', '1', 'HEAD']);
+      const hasLocalChanges = (await getExecOutput('git', ['status', '-s'], execOptions)).stdout.split('\n').length > 0;
```

Comment:
Does the output of `git` always `\n` for EOL or is it OS relative?

```suggestion
      const hasLocalChanges = (await getExecOutput('git', ['status', '-s'], execOptions)).stdout.split(EOL).length > 0;
```

Reply:
it doesn't matter to count the number of lines

Reply:
good point